client/blocks/get-apps/desktop-download-card.jsx

```diff
@@ -53,32 +61,62 @@ class DesktopDownloadCard extends Component {
 		}
 	}
 
+	getPlatformImage( platform ) {
+		switch ( platform ) {
+			case 'MacIntel':
+			case MAC_LINK:
+				return <SVGIcon name="apple-logo" size="24" icon={ Apple } />;
+			case 'Linux i686':
+			case 'Linux i686 on x86_64':
+			case LINUX_TAR_LINK:
+			case LINUX_DEB_LINK:
+				return <SVGIcon name="linux-logo" size="24" icon={ Linux } />;
+			case WINDOWS_LINK:
+			default:
+				return <SVGIcon name="windows-logo" size="24" icon={ Windows } />;
```

Comment:
It seems these icons are decorative in their use. WDYT about hiding them from screen readers? Currently, focusing on the Apple image results in merely "image" read aloud. The other icons are only used nested within anchors, so they are not focusable, but it may make sense to hide them for consistency given they do not have `title` attributes. 

For context, CSS-Tricks has a [helpful article](https://css-tricks.com/accessible-svgs/#aa-example-2-standalone-icon-decorative) on this subject. 

```suggestion
				return <SVGIcon aria-hidden="true" name="apple-logo" size="24" icon={ Apple } />;
			case 'Linux i686':
			case 'Linux i686 on x86_64':
			case LINUX_TAR_LINK:
			case LINUX_DEB_LINK:
				return <SVGIcon aria-hidden="true" name="linux-logo" size="24" icon={ Linux } />;
			case WINDOWS_LINK:
			default:
				return <SVGIcon aria-hidden="true" name="windows-logo" size="24" icon={ Windows } />;
```